### PR TITLE
opentelemetry-collector-releases-45 adding licences in containers

### PR DESCRIPTION
**Pull request by shree007**
Description: Adding all licences in resulting containers

Link to tracking Issue: https://github.com/open-telemetry/opentelemetry-collector-releases/issues/45

### CODE REVIEW

**Comment by shree007** [comment]
@jpkrohling Please review.

File: Makefile [hunk 1]
```diff
@@ -75,3 +76,21 @@ push-tag:
 	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
 	@echo "Pushing tag ${TAG}"
 	@git push git@github.com:open-telemetry/opentelemetry-collector-releases.git ${TAG}
+
+.PHONY: go-licenses
+go-licenses:
+	 go get github.com/google/go-licenses && go build github.com/google/go-licenses && mv go-licenses /usr/bin/
```
Comment (jsuereth) [inline Makefile:82]: I'm not sure we want to "contiminate" user linux distros by installing go-licenses into /usr/bin.

Is there anyway to install this somewhere local-ish and reference that local version later?

Comment (shree007) [reply Makefile:82]: @jsuereth I am not aware that.

Comment (jpkrohling) [reply Makefile:82]: It should not install under /usr/bin . Otherwise, we'd require sudo to run make, which is not acceptable.

Comment (shree007) [reply Makefile:82]: @jpkrohling @jsuereth Strongly Agree. Let me re-look into it. How may I make it without putting in /usr/bin

Comment (jpkrohling) [reply Makefile:82]: You can just install it in the home directory, possibly with `go install github.com/google/go-licenses`. We have something similar for other tools in this repository for ocb, and for other tools as part of the operator repository.

Comment (shree007) [reply Makefile]: @jsuereth @jpkrohling @anuraaga Please review now. I have put go-licenses install in OTELCOL_BUILDER_DIR.

**Review by jsuereth — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -75,3 +76,21 @@ push-tag:
 	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
 	@echo "Pushing tag ${TAG}"
 	@git push git@github.com:open-telemetry/opentelemetry-collector-releases.git ${TAG}
+
+.PHONY: go-licenses
+go-licenses:
+	 cd ${OTELCOL_BUILDER_DIR} && go install github.com/google/go-licenses@latest
```
Comment (bogdandrutu) [inline Makefile:82]: Consider to use a fixed version instead of latest to have deterministic builds.

Comment (shree007) [reply Makefile]: @bogdandrutu I have added latest version `v1.2.0`, Please review

**Comment by jpkrohling** [comment]
The build seem to be failing.

**Comment by shree007** [comment]
@jpkrohling Do I need to fix Build ?

**Comment by jpkrohling** [comment]
Take a look at why it's failing. If you can determine the cause and it's not related to this PR, then we might just want to restart the build.

**Comment by shree007** [comment]
I checked locally while executing 'make ci', In pipeline not able to 'cd /home/runner/bin'

**Comment by bogdandrutu** [comment]
Can you "rebase" to re-trigger the checks? Reason I think this is too old so we cannot restart.

**Comment by bogdandrutu** [comment]
```
Run make ci
go: downloading github.com/goreleaser/goreleaser v1.9.2
go: downloading github.com/goreleaser/nfpm/v2 v2.16.0
go: downloading gopkg.in/yaml.v2 v2.[4](https://github.com/open-telemetry/opentelemetry-collector-releases/runs/7504115447?check_suite_focus=true#step:4:5).0
go: downloading github.com/alecthomas/jsonschema v0.0.0-20211209230136-e2b41affa[5](https://github.com/open-telemetry/opentelemetry-collector-releases/runs/7504115447?check_suite_focus=true#step:4:6)c1
go: downloading github.com/apex/log v1.9.0
go: downloading github.com/iancoleman/orderedmap v0.2.0
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/goreleaser/fileglob v1.3.0
go: downloading github.com/gobwas/glob v0.2.3
cd /home/runner/bin && go install github.com/google/go-licenses@v1.2.0
/bin/sh: 1: cd: can't cd to /home/runner/bin
make: *** [Makefile:[8](https://github.com/open-telemetry/opentelemetry-collector-releases/runs/7504115447?check_suite_focus=true#step:4:9)2: go-licenses] Error 2
Error: Process completed with exit code 2.
```

**Comment by jpkrohling** [comment]
@shree007, once CI is passing, ping me and I'll review this again.

**Comment by shree007** [comment]
Approval is required to proceed

File: Makefile [hunk 3]
```diff
@@ -75,3 +76,21 @@ push-tag:
 	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
 	@echo "Pushing tag ${TAG}"
 	@git push git@github.com:open-telemetry/opentelemetry-collector-releases.git ${TAG}
+
+.PHONY: go-licenses
+go-licenses:
+	 chmod -R 775 ${OTELCOL_BUILDER_DIR} && cd ${OTELCOL_BUILDER_DIR} && go install github.com/google/go-licenses@v1.2.0
```
Comment (jpkrohling) [inline Makefile:82]: `chmod -R 775` is never a good solution. Why do you need this?

**Review by jpkrohling — CHANGES_REQUESTED** [review]

**Comment by shree007** [comment]
There is permission denied while doing 'cd folder-path' so i thought to do
something like that.

When I running locally there is nothing 'permission denied' issue and i
compare to both path.

**Comment by jpkrohling** [comment]
`chmod -R 755` will recursively change everything in the build dir to allow everyone to read and write everywhere. There has to be a better way to achieve what you need.

**Comment by shree007** [comment]
Let me try though.

**Comment by jpkrohling** [comment]
This PR has been around for quite a long time. I'm closing it.

**Pull request closed without merging**